core-java-modules/core-java-string-algorithms-3/src/test/java/com/baeldung/repeatedcharstring/RepeatedCharacterStringUnitTest.java

```diff
@@ -0,0 +1,136 @@
+package com.baeldung.repeatedcharstring;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.nio.CharBuffer;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Strings;
+
+import static ch.qos.logback.core.CoreConstants.EMPTY_STRING;
+import static java.util.stream.Stream.generate;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class RepeatedCharacterStringUnitTest {
+
+    @Test
+    public void givenString_whenStringBufferUsed_thenStringCreated() {
+        String exampleString = "aaaaaaa";
```

Comment:
Let's extract this to a `private static final String EXAMPLE_STRING` constant, rather than repeating it in each test case.

Reply:
resolved